packages/js/components/src/experimental-tree-control/stories/index.tsx

```diff
@@ -36,6 +36,36 @@ export const SimpleTree: React.FC = () => {
 	);
 };
 
+function isItemExpanded( item: LinkedTree, text: string ) {
```

Comment:
Same comment here re: `is/should`.  I also think `text` might be better as `query` if I'm understanding what this is correctly (user search input).

```suggestion
function shouldItemExpand( item: LinkedTree, query: string ) {
```